susemanager-frontend/susemanager-nodejs-sdk-devel/susemanager-nodejs-sdk-devel.spec

```diff
@@ -14,7 +14,7 @@
 #
 
 Name:           susemanager-nodejs-sdk-devel
-Version:        1.0.10
+Version:        4.1.0
```

Comment:
Hmm, why was it 1.0.10 and now we switch to 4.1?

Reply:
Well I maintained the same versioning when I migrated it from obs to git, but we can change it to 4 sure, it doesn't have any impact. This package is not even installed, it's only used during build time

Reply:
Then I'd say: let's make it easier for the future.

@mcalmer, anything against it?

Reply:
Ok, in the past it was not part of the git. Yes, we can go for version 4.1